### DISCUSSION TITLE
Alerts doesnt come up in Import Node

### DISCUSSION THREAD

**Discussion by ekky19** [Security-Onion-Solutions/securityonion#8151] [Unsupported Versions]
I can see the logs in dashboard, hunt, or kibana, but nothing comes up in **Alerts** in the import node. This really makes my life very hard. I couldn't find a solution, unfortunately. it is just a fresh installation Version: 2.3.130 on centos 7. What could be wrong?

Here is the hardware info:
![image](https://user-images.githubusercontent.com/6872571/174441541-335b6183-47ee-4156-ad41-5a4a3a11f0b3.png)

Alert:
![image](https://user-images.githubusercontent.com/6872571/174441600-a9c1eeac-e0a0-4994-b9c2-fd56917d3c5f.png)

Dashboard:
![image](https://user-images.githubusercontent.com/6872571/174441634-015bd5ad-cb6b-42c7-9c0a-104f28140332.png)

**Comment by dougburks** [comment]
Please see the Troubleshooting Alerts section of the documentation:
https://docs.securityonion.net/en/2.3/suricata.html#troubleshooting-alerts
